### ISSUE TITLE
Add owners of each dandisets as contributors

### ISSUE THREAD

**Issue by waxlamp** [dandi/dandiarchive#755]
This is a one-time fix operation that depends on #425 being solved first.

The fix should, for each dandiset, add any owners as contributors if they are not already present in the contributor list.

**Comment by sanuann** [comment]
is this issue resolved?

**Comment by waxlamp** [comment]
I don't think so: look at this dandiset for example https://gui.dandiarchive.org/#/dandiset/000050.

New dandisets do have the creator added as a contributor, and I don't think adding an owner to a dandiset adds them as a contributor (nor am I sure that we want that to happen--@satra?).

This issue is not really about those policies though--this has to do with adding owners as contributors to current draft dandisets. We need some planning over exactly what should be done, and then the work will be done as a maintenance script on the Django side of things (at least, that's my understanding).

@dchiquito, I'm assigning this issue to you, but as I mentioned it still requires a bit of planning before we can really do anything.

**Comment by satra** [comment]
i think new dandisets add the owner/creator as contributor + contactperson and i think that is correct.

regarding older dandisets, we should do the same for all dandisets missing a contributor.

**Comment by waxlamp** [comment]
Do we add *all* owners as contributors? Which one do we mark as the contact?

**Comment by satra** [comment]
for the pass through old dataset, i would say add all and make all `contactPerson`. they can change it if they want. this should only be done for those dandisets that have no contributors.
